.ci/e2eKibana.groovy

```diff
@@ -46,15 +46,57 @@ pipeline {
     string(name: 'kibana_pr', defaultValue: "master", description: "PR ID to use to build the Docker image. (e.g 10000)")
   }
   stages {
+    stage('Initialize'){
+      agent { label 'ubuntu-20 && immutable' }
+      environment {
+        HOME = "${env.WORKSPACE}/${BASE_DIR}"
+      }
+      steps {
+        deleteDir()
+        checkPermissions()
+        setEnvVar('DOCKER_TAG', getDockerTagFromPayload())
+      }
+    }
     stage('Process GitHub Event') {
-      agent { label 'ubuntu-20' }
       environment {
         HOME = "${env.WORKSPACE}/${BASE_DIR}"
         PATH = "${env.HOME}/bin:${env.HOME}/node_modules:${env.HOME}/node_modules/.bin:${env.PATH}"
       }
+      parallel {
+        stage('AMD build') {
+          agent { label 'ubuntu-20' }
+          steps {
+            buildKibanaPlatformImage('amd64')
+          }
+        }
+        stage('ARM build') {
+          agent { label 'arm' }
+          steps {
+            buildKibanaPlatformImage('arm64')
+          }
+        }
+      }
+    }
+    stage('Push multiplatform manifest'){
```

Comment:
NIT: you can create a meta-stage with one node allocation that will be used in this stage and the following one, in case you'd like to reuse the same worker and reduce the allocation time

Reply:
@v1v something like in https://github.com/elastic/e2e-testing/pull/1245/commits/2f8a24c0c5861e05fc28d0a1624ddcb99f0a0552?